### ISSUE TITLE
Frontend Development Server Failure 

### ISSUE THREAD

**Issue by Yicong-Huang** [ISG-ICS/Wildfires#3]
**Describe the bug**
Temporarily, we are using the Angular Development Server to deploy the demo. However, it will fail about 2 days. 

**To Reproduce**
Launch the Angular server on `wildfires.ics.uci.edu`, for more than 2 days.

**Expected behavior**
Either solution would benefit us:
1. Fix the issue so that the development server will not go down after about 2 days.
2. Use production mode to generate the html, css, js. and use backend flask server to serve.

**Error Messages**
```
URIError: Failed to decode param '/%NETHOOD%/'
    at decodeURIComponent (<anonymous>)
    at decode_param (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/layer.js:172:12)
    at Layer.match (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/layer.js:123:27)
    at matchLayer (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:574:18)
    at next (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:220:15)
    at expressInit (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/middleware/init.js:40:5)
    at Layer.handle [as handle_request] (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:317:13)
    at /extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:335:12)
    at next (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:275:10)
    at query (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/middleware/query.js:45:5)
    at Layer.handle [as handle_request] (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:317:13)
    at /extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/extra/yicongh10/demo/Wildfires/frontend/node_modules/express/lib/router/index.js:335:12)
events.js:170
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:171:27)
Emitted 'error' event at:
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at processTicksAndRejections (internal/process/task_queues.js:81:17)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wildfires-frontend@0.0.0 start: `ng serve --host 0.0.0.0 --port 2333 --disableHostCheck`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wildfires-frontend@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/yicongh1/.npm/_logs/2019-10-05T05_08_45_087Z-debug.log
yicongh1@cloudberry05 22:08:45 /extra/yicongh10/demo/Wildfires/frontend
```

**Desktop (please complete the following information):**
 - OS: CentOS 7

**Comment by AC-Unicorn** [comment]
Do we have a log that contains all Http requests

**Comment by Yicong-Huang** [comment]
partially fixed by #8, close this issue for now.
